backend/tournesol/views/preview.py

```diff
@@ -0,0 +1,50 @@
+from PIL import Image, ImageDraw, ImageFont
+import requests
+from io import BytesIO
+from django.http import HttpResponse, FileResponse
+from django.utils.decorators import method_decorator
+from drf_spectacular.utils import OpenApiTypes, extend_schema
+from rest_framework.authentication import SessionAuthentication
+from rest_framework.permissions import AllowAny, IsAdminUser, IsAuthenticated
+from rest_framework.views import APIView
+
+POPPINS_FONT = ImageFont.truetype("../frontend/public/fonts/Poppins-Bold.ttf", 40)
+
+
+class DynamicWebsitePreview(APIView):
+    permission_classes = []
+
+    @extend_schema(
+        description="preview the ",
+        responses={200: OpenApiTypes.BINARY},
+    )
+    def get(self, request):
+        default_preview = open("../frontend/public/tournesol_screenshot_og.png", "rb")
+        response = FileResponse(default_preview, content_type="image/png")
+        return response
```

Comment:
The same for this image path. Either move the file to "backend" or use a HTTP redirection towards a frontend URL (if the frontend should remain responsible for this default preview image).

Reply:
Done